build_debian.sh

```diff
@@ -158,6 +158,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT useradd -G sudo,docker $USERNAME -c "$DEFAUL
 ## Create password for the default user
 echo $USERNAME:$PASSWORD_ENCRYPTED | sudo LANG=C chroot $FILESYSTEM_ROOT chpasswd -e
 
+## Create remote user
```

Comment:
1. It is confusing for all other users not using TACACS+.
2. Do we need to install rbash on SONiC host?
3. The username, uid, gid are hard-coded. Are they wellknown in TACACS+?